src/components/room.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from "react";
+import io from "socket.io-client";
+import RoomForm from "./roomForm";
+import VideoPlayer from "./videoPlayer";
+import { VIDEO_PLAYER_ACTIONS } from "./../constants/videoPlayerActions";
+
+class Room extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      socket: null,
+      url: null
+    };
+    this.videoPlayerRef = React.createRef();
```

Comment:
Change this name as it's misleading as in component video player it has the same name.